src/components/BackLink/BackLink.tsx

```diff
@@ -9,14 +9,15 @@ import {DefaultEventNames, Tabbable} from '../../models';
 
 export type Theme = 'default' | 'special';
 
-export interface BackLinkProps extends Tabbable {
+export interface BackLinkProps<T = HTMLElement> extends Tabbable {
     url: string;
     title: ReactNode;
     theme?: Theme;
     size?: ButtonSize;
     className?: string;
     shouldHandleBackAction?: boolean;
     onClick?: () => void;
+    extraProps?: HTMLProps<T>;
```

Comment:
maybe `htmlProps` or `htmlElementProps` would be more obvious?

Reply:
The name `extraProps` is the commonly used name in the uikit, so I think this variant is appropriate here